packages/infra/lib/patient-import-nested-stack.ts

```diff
@@ -13,19 +13,21 @@ import { createLambda } from "./shared/lambda";
 import { LambdaLayers } from "./shared/lambda-layers";
 import { createQueue } from "./shared/sqs";
 
+const waitTimePatientCreateInSeconds = 300; // 5min
+const waitTimePatientQueryInSeconds = 0;
+
 function settings() {
-  // TODO 2330 CUSTOMIZE THIS
-  // TODO 2330 CUSTOMIZE THIS
-  // TODO 2330 CUSTOMIZE THIS
-  const waitTime = Duration.seconds(20);
   const fileImportLambdaTimeout = Duration.minutes(15).minus(Duration.seconds(5));
   const fileImport = {
     name: "PatientImportFile",
     entry: "patient-import-file",
     lambdaMemory: 2048,
     lambdaTimeout: fileImportLambdaTimeout,
   };
-  const patientCreateLambdaTimeout = Duration.minutes(5).minus(Duration.seconds(5));
+  const waitTimePatientCreate = Duration.minutes(waitTimePatientCreateInSeconds);
+  const patientCreateLambdaTimeout = Duration.seconds(waitTimePatientCreateInSeconds + 30).minus(
+    Duration.seconds(5)
+  ); // 25secs for processinng
```

Comment:
for later: simplify this

Reply:
Will handle in v1.

Reply:
Done via patch https://github.com/metriport/metriport/pull/2862